pallets/liquidity-rewards/src/lib.rs

```diff
@@ -267,7 +193,7 @@ pub mod pallet {
 	#[pallet::hooks]
 	impl<T: Config> Hooks<T::BlockNumber> for Pallet<T> {
 		fn on_initialize(current_block: T::BlockNumber) -> Weight {
-			let ends_on = EndOfEpoch::<T>::get().0;
+			let ends_on = EndOfEpoch::<T>::get();
```

Comment:
I guess in the initial call it will return the BlockNumber default of its primitive type, which should be 0, so the following comparison should be happy

Reply:
Yes, default `BlockNumber` is  `zero()`